pytorch_lightning/profiler/pytorch/pytorch_kineto.py

```diff
@@ -0,0 +1,338 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Profiler to check if there are any bottlenecks in your code."""
+import inspect
+import os
+import time
+from collections import defaultdict
+from pathlib import Path
+from typing import Any, Dict, List, Optional, Set, Type, TYPE_CHECKING, Union
+
+import torch
+from torch.autograd.profiler import record_function
+
+from pytorch_lightning.profiler.base import BaseProfiler
+from pytorch_lightning.profiler.pytorch.pytorch_utils import RegisterRecordFunction
+from pytorch_lightning.utilities import rank_zero_warn
+from pytorch_lightning.utilities.exceptions import MisconfigurationException
+from pytorch_lightning.utilities.imports import _KINETO_AVAILABLE
+
+if TYPE_CHECKING:
+    from torch.autograd.profiler import EventList
+
+    from pytorch_lightning.core.lightning import LightningModule
+
+if _KINETO_AVAILABLE:
```

Comment:
There might not be a need for `_KINETO_AVAILABLE` as it is imported only with `_TORCH_GREATER_EQUAL_1_10 `

Reply:
good suggestion